app/assets/javascripts/comfortable_mexican_sofa/admin/modules/CategoryExpand.js

```diff
@@ -0,0 +1,71 @@
+define(['jquery','DoughBaseComponent'], function ($, DoughBaseComponent) {
+'use strict';
+
+  var defaultConfig = {
+    categoriesCharacterCount: 20
+  };
+  var CategoryExpandProto;
+
+  function CategoryExpand($el, config) {
+    CategoryExpand.baseConstructor.call(this, $el, config, defaultConfig);
+  }
+
+  DoughBaseComponent.extend(CategoryExpand);
+
+  CategoryExpand.componentName = 'CategoryExpand';
+
+  CategoryExpandProto = CategoryExpand.prototype;
+
+  CategoryExpandProto.init = function(initialised) {
+
+    this.addExpanders();
+
+    $('#filter_form').on('ajax:complete', $.proxy(function() {
+      this.addExpanders();
+    }, this));
+
+    this._initialisedSuccess(initialised);
+      return this;
+    };
+
+    CategoryExpandProto.addExpanders = function(){
+
+    var self = this;
+
+    $('.js-filter-pages td:nth-child(3)').addClass('categories');
```

Comment:
@johnplayer1982 can do that here https://github.com/moneyadviceservice/cms/blob/master/app/views/comfy/cms/pages/_page.html.haml